cmd/containerd/command/service_windows.go

```diff
@@ -155,38 +154,14 @@ func registerService() error {
 	}
 	defer s.Close()
 
-	// See http://stackoverflow.com/questions/35151052/how-do-i-configure-failure-actions-of-a-windows-service-written-in-go
-	const (
-		scActionNone    = 0
-		scActionRestart = 1
-
-		serviceConfigFailureActions = 2
+	return s.SetRecoveryActions(
+		[]mgr.RecoveryAction{
+			{Type: mgr.ServiceRestart, Delay: 15 * time.Second / time.Millisecond},
```

Comment:
`Delay` is a `time.Duration`, and gets converted to milliseconds by `SetRecoveryActions`, so we should just pass `15*time.Second` here.

Reply:
Ooof! Good catch 😬 🤦‍♂️  It crossed my mind, but then went off to check _"Did we change this timeout? "_ (to go and find https://github.com/containerd/containerd/commit/cee19d1e6eafc38b90dd2a4f8ed442784cec3169).

Either way; fixed it, thanks!